src/Styleguide/Components/Tabs.tsx

```diff
@@ -49,12 +51,17 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
   }
 
   renderTab = (tab, index) => {
-    const { name } = tab.props
+    const { name, children, ...preloadProps } = tab.props
+    const buttonContent = Object.keys(preloadProps).length ? (
```

Comment:
Maybe could use more robust logic to see if it's a preload or not.

Reply:
Can you add a `FIXME` here with this comment? At some point we can check the shape of `preloadProps` against the interface.